cypress/integration/all/app/dashboard.spec.ts

```diff
@@ -28,4 +34,25 @@ describe("Dashboard", () => {
       cy.getAttached(".operating-systems").should("exist");
     });
   });
+
+  describe("Activity Card", () => {
+    beforeEach(() => {
+      cy.loginWithCySession();
+    });
+
+    it("displays activity when editing agent options", () => {
+      AppSettingsPage.visitAgentOptions();
+      AppSettingsPage.editAgentOptionsForm("test:");
```

Comment:
Maybe add a waitUntil the confirmation flash message appears? It may be that you are submitting the change, then going to the dashboard to look for the activity feed before the backend has created it.